tools/quicktype-wrapper/src/index.ts

```diff
@@ -91,27 +90,29 @@ if (!module.parent) {
     // Loop through every path
     const pathPromises = absPaths.map(async (f: string, i: number) => {
       const file = await readFileSync(f) + '';
-      const relPath = relPaths[i];
-      const typeName = JSON.parse(file).name;
+      const relPath = relPaths[i]; // e.g. /google/events/cloud/pubsub/MessagePublishedData.json
```

Comment:
I'm confused by this being called relPath (which I assume to mean "relative") but with an example starting with "/" which sounds absolute. Is the example correct, or would it actually start with google?

Reply:
Good point. Yeah it's relative, it's all the stuff after "googleapis/google-cloudevents/jsonschema". Naming is hard. Probably should rename to something else or change the logic. Path separators are a bit tedious to keep track of with all these generators.

Reply:
Given the example, this might be more readable with a topical name.

E.g., pathToSchema